packages/compose/src/hooks/use-focus-on-mount/README.md

```diff
@@ -0,0 +1,28 @@
+`useFocusOnMount`
+=================
+
+Hook used to focus the first tabbable element on mount.
+
+## Return Object Properties
+
+### `ref`
+
+- Type: `Function`
```

Comment:
So this is incorrect, right? An object ref is returned.

Reply:
Oh right! I  guess it should just say "Element ref". We don't really care whether it's a function or object.